scripts/linters/general_purpose_linter.py

```diff
@@ -570,6 +558,36 @@ def check_newline_at_eof(self):
         return concurrent_task_utils.TaskResult(
             name, failed, error_messages, error_messages)
 
+    def check_disallowed_flags(self):
+        """This function is used to disallow flags."""
+        name = 'Disallow flags'
+        disallow_flag = (
+            'eslint-disable-next-line oppia/no-bypass-security-phrase')
+        error_messages = []
+        files_to_lint = self.all_filepaths
+        failed = False
+        excluded_files = (
+            warranted_angular_security_bypasses
+            .EXCLUDED_BYPASS_SECURITY_TRUST_FILES)
+        allowed_files = ''
+        for filepath in files_to_lint:
+            for excluded_file in excluded_files:
+                if excluded_file in filepath:
+                    allowed_files = filepath
+            if not filepath.endswith('.ts') or filepath == allowed_files:
+                continue
+            file_content = self.file_cache.read(filepath)
+
+            if disallow_flag in file_content:
+                error_message = (
+                    '%s --> The no-bypass-security-phrase flag is only '
+                    'expected to be use in the file mentioned in '
+                    'warranted_angular_security_bypasses.py' % filepath)
```

Comment:
Change this error message, try to use something more descriptive that clearly describe what to do in case someone faces this lint error like : `Please do not use 'no-bypass-security-phrase' flag. It is only expected to be used in files listed in warranted_angular_security_bypasses.py`

Reply:
Done!